server/schemas/typeDefs.js

```diff
@@ -1,37 +1,40 @@
 const { gql } = require('apollo-server-express');
 
 const typeDefs = gql`
-  type Category {
-    _id: ID
-    name: String
-  }
-
-  type Product {
-    _id: ID
-    name: String
-    description: String
-    image: String
-    quantity: Int
-    price: Float
-    category: Category
+  type User {
+    _id: ID!
+    firstName: String!
+    lastName: String!
+    email: String!
+    pets: [Pet]
   }
 
-  type Order {
-    _id: ID
-    purchaseDate: String
-    products: [Product]
+  type Pet {
+    _id: ID!
+    name: String!
+    birthday: Int!
+    pet: String!
```

Comment:
pet property in a type name PET is incorrect. The property should be 'name'.

Reply:
As I mentioned, the per property is for the type of pet (Dog, Cat, etc.)

Reply:
I can change it to petType if you prefer

Reply:
same as before . I am talking about the pet's name.  The same concept applies since the Schema is Pet. It looks confusing to use Pet.pet  instead of Pet.kind, Pet.petKind, Pet.petType, or Pet.type.

Reply:
Pricisila, the name is included in the schema, see below, is the first property that I added to it.

const petSchema = new Schema({
  name: {
    type: String,
    required: true

Reply:
PetType, should be its Own Schema, so we can easily add PetTypes. Which will be displayed in the combo box in the registration page.

Reply:
Ok, No problem, I will add a PetType model; note that that was not in the models that we discussed, but I agree that it can be its own model.